README.md

```diff
@@ -1,2 +1,240 @@
-# space-travelers
-React Redux Project with group
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <h3><b>Space Travelers</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[space-travelers\] ](#-space-travelers-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment ](#deployment-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [space-travelers] <a name="about-project"></a>
+
+
+
+**[space-travelers]** is a project related to space mission. Users can reserve a rocket, and can join a mission. User can also be able to see data of reserved rockets, and joined missions in My profile tab. It is a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions using SpaceX API.
+
+## 🛠 Built With <a name="built-with"></a>
+JavaScript
+React
+Redux
+Jest
+HTML
+CSS
+APIs
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML, CSS, JavaScript, React, Redux, Jest, APIs</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://localhost:3000/">Dev Server</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Pakage Manager</summary>
+  <ul>
+    <li><a href="#">NPM</a></li>
+  </ul>
+</details>
+<details>
+  <summary>API</summary>
+  <ul>
+    <li><a href="#">External API</a></li>
```

Comment:
- [ ] Kindly fix all theses broken links that redirected me to wrong page, this will ensure that your README file is engaging and professional. 👍

Reply:
I have done the changes you suggested. Thanks a lot for your kind feedback.